frb_example/pure_dart/dart/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   js: ^0.6.5
   test_core: ^0.4.16
   test: ^1.21.4
-  uuid: ^3.0.6
+  uuid:
```

Comment:
Hmm this does not sound very good in the final code :/

Reply:
Ideally we have one unspecified dep in order to test this, do you have one in mind? I could also add a random, simple dep to dev_dependencies

Reply:
Maybe add a (rust) unit test

Reply:
Alright sounds good, lemme do that.